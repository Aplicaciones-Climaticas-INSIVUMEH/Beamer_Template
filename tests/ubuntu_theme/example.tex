\documentclass{beamer}
\usepackage[utf8]{inputenc}
\usepackage[T1]{fontenc}
\title{Por que pondrías eso?}
\date[ISPN ’80]{a}
\author[Cuzco]{Josué Cuzco \texttt{jdcuzco@insivumeh.gob.gt}}

\usetheme{ubuntu}

\begin{document}

\begin{frame}
\titlepage
\end{frame}

\begin{frame}
    \frametitle{hol}

    hola

\end{frame}

\section{Hola}

\begin{frame}
    \frametitle{hol}

    hola

\end{frame}

\end{document}