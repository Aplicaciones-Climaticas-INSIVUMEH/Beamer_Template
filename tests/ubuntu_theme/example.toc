\beamer@sectionintoc {1}{Hola}{3}{0}{1}
