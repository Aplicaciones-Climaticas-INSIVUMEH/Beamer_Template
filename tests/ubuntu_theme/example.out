\BOOKMARK [2][]{Outline0.1}{\376\377\000H\000o\000l\000a}{}% 1
